app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -802,11 +802,7 @@ export default {
       padding: 8px 24px;
     }
     .btn-primary.blazing-battle {
-      white-space: pre-wrap; /* CSS3 */    
-      white-space: -moz-pre-wrap; /* Mozilla, since 1999 */
-      white-space: -pre-wrap; /* Opera 4-6 */    
-      white-space: -o-pre-wrap; /* Opera 7 */    
-      word-wrap: break-word; /* Internet Explorer 5.5+ */
+      white-space: pre-wrap;
```

Comment:
I was also suggesting that this could just be `normal` instead of this strange `pre-wrap` whitespace, as this is not a `pre` element, and `normal` seems to work.

Reply:
Updated to `white-space: normal;`